draft-ietf-tls-tls13.md

```diff
@@ -2538,17 +2503,27 @@ not the the 0-RTT cryptographic transforms, including:
 What's here now needs a lot of cleanup before it is clear
 and correct.]]
 
-[[TODO: We should really allow early_data to be used with
-PSKs. In order to make this work, we need to either:
 
-(a) explicitly signal the entire cryptographic parameter set
-(b) tie it to the PSK identifier (as is presently done in the
-    known_configuration extension).
 
-These two should match.
-]]
+##### Cryptographic Determining Parameters
 
+In order to allow the server to decrypt 0-RTT data, the client
+needs to provide enough information to allow the server to
+decrypt the traffic without negotiation. This is accomplished
+by having the client indicate the "cryptographic determining
+parameters" in its ClientHello, which are necessary to decrypt
+the client's packets. This includes the following
+values 
 
+- The cipher suite identifier.
```

Comment:
Needs an extra newline between these two bullet points to match the style from the rest of the document.